test/execution/execution.test.ts

```diff
@@ -25,13 +28,15 @@ suite('Gauge Execution Tests', () => {
         assertStatus(status);
     }).timeout(20000);
 
-    test('should execute given scenario', async () => {
+    test.only('should execute given scenario', async () => {
```

Comment:
is `only` intended?

Reply:
Yes, I was trying to debug this particular test. Now every thing passing so I have removeded it.